js/controller.js

```diff
@@ -2159,7 +2159,7 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
         this.mb.publish(OO.EVENTS.REQUEST_PREVIOUS_VIDEO);
       } else {
         this.updateSeekingPlayhead(0);
-        this.mb.publish(OO.EVENTS.REPLAY);
```

Comment:
REPLAY event is not published?

Reply:
Not anymore, it will just perform a seek. I think it makes sense since `REPLAY` currently happens only after the whole video has been watched, so rewinding is not the same thing. If you think this should be fired I can look for another solution though.

Reply:
I think that makes sense. For analytics I think we want to separate out when the user wants to watch the content again versus rewinding halfway through.